File: src/Generated20260217.go
package main

import "fmt"

type Generated20260217 struct {
    // Generated struct 20260217_061359_829705
    Value int
    Created string
}

func (c *Generated20260217) Process() int {
    result := c.Value
    for i := 0; i < 8; i++ {
        result += i * 5
    }
    return result
}

func main() {
    obj := Generated20260217{Value: 955, Created: "20260217_061359_829705"}
    fmt.Println(obj.Process())
}

File: src/Generated20260217.cpp
#include <iostream>
#include <string>
using namespace std;

class Generated20260217 {
public:
    // Generated class 20260217_061527_119947
    int value = 363;
    string created = "20260217_061527_119947";
    
    int process() {
        int result = value;
        for (int i = 0; i < 36; i++) {
            result += i * 9;
        }
        return result;
    }
};

int main() {
    Generated20260217 obj;
    cout << obj.process() << endl;
    return 0;
}